### DISCUSSION TITLE
How to efficiently delete many downloaded maps (here: hillshading and slopes)?

### DISCUSSION THREAD

**Discussion by dreirund** [osmandapp/OsmAnd#17941] [Q&A]
Ahoj,

recent OsmAnd update now uses single heightmap data for both hillshading and slope display.

But I still have many old, now obsolete, hillshading and slope files downloaded.

What is an easy way to remove all of them?

When I go to "Download maps" -> "LOCAL", I have to scroll down to find the downloaded hillshading and slope maps (scattered in between online maps). Then I can delete one. Then I start at the top of the screen, having to scroll down a lot again.

That is not a viable solution for many tens, maybe hundred, of such files.

Regards!

**Comment by mikehgentry** [comment]
On Android? I'd export your profiles and any other data you want to keep (just in case), then connect the phone to a computer and delete them through the filesystem.

(it doesn't cause problems with voice packs, and I doubt it does with maps either. But I don't think I've actually tried it, hence the profile backup)

**Reply by dreirund** [reply]
> *On Android? I'd export your profiles and any other data you want to keep (just in case), then connect the phone to a computer and delete them through the filesystem.*

So no easy way from within OsmAnd? &rarr; Made [&nearr;&hairsp;a feature request](https://github.com/osmandapp/OsmAnd/issues/17952)!
